Dynatrace/dynatrace-operator
# Add monitored-entity reconciler to logmodule reconciler

## Description
[DAQ-1397](https://dt-rnd.atlassian.net/browse/DAQ-1397)

Both `KubernetesClusterMEID` and `KubernetesClusterName` are a hard requirement for the deployment of the standalone logmodule.

So we add the `monitoredentities.Reconciler` to the logmodule reconcile, the ensure that we have it available.
- If we still don't have the `KubernetesClusterMEID` and `KubernetesClusterName` then we log it and then end the logmodule reconcile early.
   - should we error instead of log? 

## How can this be tested?

Deploy a dynakube with `logModule.enable: true` in 2 scenarios:
1. When there is an ME on your tenant => everything should deploy fine (daemonset appears)
2. When there is **NO** ME on your tenant => should see in the logs that something is not good (daemonset does NOT appear)

[DAQ-1397]: https://dt-rnd.atlassian.net/browse/DAQ-1397?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
